.changeset/hungry-cups-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": patch
```

Comment:
Blocking merge until confirmation

Reply:
It seems this PR should not close: https://github.com/FuelLabs/fuels-ts/issues/2898; just mention it. The release should probably not say it's also fixed, maybe patched, or perhaps we can have `Release Notes` only on the final solution via https://github.com/FuelLabs/fuels-ts/pull/2901.

Reply:
Other than that, 🚢.

Reply:
🚢